modules/rules_rust/0.34.1/presubmit.yml

```diff
@@ -0,0 +1,647 @@
+---
+aspects_flags: &aspects_flags
```

Comment:
This collection of checks looks very heavy. Most Bzlmod-related failures can already be verified by running just the analysis phase. Given the already heavy load on the bazelbuild CI, would it be possible to cut this down and/or sprinkle in uses of `--nobuild`?

CC @meteorcloudy who knows the state of the CI system better

Reply:
Thanks! Yes, I totally agree with @fmeum. The presubmit.yml file in the BCR is not meant to run all the integration/unit tests in your normal CI pipeline. It should be a few integration tests that cover the most common usages of this module. Ideally, by including a [test module](https://github.com/bazelbuild/bazel-central-registry/tree/main/docs#test-module)

Reply:
I will propose a simpler pre-submit check as the full set of checks will have already run for the PRs.

Reply:
I put up [a PR](https://github.com/bazelbuild/rules_rust/pull/2347) which introduces a `rules_bazel_integration_test` that builds and runs a hello world using `rules_rust`.

Reply:
Fixed by https://github.com/bazelbuild/rules_rust/pull/2347.